src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/listeners/PSPlayerListener.java

```diff
@@ -541,6 +541,24 @@ public void onPlayerInteract(PlayerInteractEvent event) {
                 }
             }
         }
+        
+        // -------------------------------------------------------------------------------- trying to spawn mobs/animals with spawn egg
+
+		if (is.getType().name().contains("_EGG")) {
```

Comment:
It feels like there ought to be a nicer way to do this- I wonder if we could do like 
`if (is.getItemMeta() instanceof SpawnEggMeta)` 
?

Reply:
Hey, yeah, that is a really nicer and smarter way! Ty! Done